tests/conftest.py

```diff
@@ -137,6 +140,14 @@ def pytest_addoption(parser):
     )
 
 
+def pytest_configure(config: _pytest.config.Config) -> None:
+    # register custom marker
+    config.addinivalue_line(
```

Comment:
Why is this done different from the `--fast` option? 

(i.e. slowtest is in the actual ini file, will that wor, if so, why isn't this like that? )

Reply:
I hadn't noticed `slowtest` was in there. I'll move it. I'll probably need to add it in `tests_common` as well.

Reply:
Applied in feb0eb5c.